lib/performance/gateway/elasticsearch.rb

```diff
@@ -5,11 +5,12 @@ def initialize(index)
     @index = index
   end
 
-  def write(data)
+  def write(key, data)
```

Comment:
I wonder if we can get rid of this class altogether. It doesn't seem to do much anymore

Reply:
Unless we're refactoring out the CA structure, this is useful to keep, as it means the `index` can be defined at instantiation in the class, and the use case doesn't need to care about the index.

Reply:
Unless we're refactoring out the CA structure, then it's good to keep this, as it means the `index` can be defined at instantiation in the task, and the use case doesn't care about the index.